src/main/scala/scalaz/netty/Client.scala

```diff
@@ -36,6 +36,7 @@ import _root_.io.netty.handler.codec._
 
 private[netty] final class Client(limit: Int) {
   // this isn't ugly or anything...
+  @volatile
```

Comment:
Ok handler refers to the server.queue. I think instead of creating queue inside the server, it should be injected into the instance instead....